_posts/2020-08-05-hiatus.md

```diff
@@ -32,17 +32,17 @@ About a year before I took the plunge and quit my job for real estate riches, a
 
 ## Traditional Foreclosure Auction
 
-With my two partners and approximately $120,000 in cash, we attended a real estate foreclosure auction. One of my partners had been in touch with the absentee owner, who said he had no intention of stopping the sale. He wished us luck. We had agreed on an absolute maximum of $108,000 as I recall. The actual number is not as important as the fact that we had a maximum, especially because neither of the other two partners were able to attend that day.
+With my two partners and approximately $120,000 in cash, I attended a real estate foreclosure auction. One of my partners had been in touch with the absentee owner, who said he had no intention of stopping the sale. He wished us luck. We had agreed on an absolute maximum of $108,000 as I recall. The actual number is not as important as the fact that we had a maximum, especially because neither of the other two partners were able to attend that day.
 
 New Hampshire foreclosures are unique in that they do not take place "on the courthouse steps" as they do in other states. Nope. NH law requires a foreclosure auction to be held _on the property being sold!_. You can imagine the spectacle this could cause, having potentially dozens of uninvited and unwelcome bidders show up unexpectedly. That is what happened to us that day.
```

Comment:
Here you talk about there being uninvited and unwelcome bidders showing up unexpectedly and that it is what happened to you that day. However, later you talk about being the only interested bidder and that it wasn't well attended.

Reply:
Yeah, I am painting an overall picture here, even though it wasn't quite as bad in this case.